.devcontainer/README.md

```diff
@@ -25,7 +29,7 @@ You can also follow these steps to open this repo in a container using the VS Co
 
 2. Open a locally cloned copy of the code:
 
-   - Clone this repository to your local filesystem.
+   - Fork and Clone this repository to your local filesystem.
```

Comment:
out of curiosity why do you need to fork for this?

Reply:
According to the contribution guidelines we should not commit to the main branch directly but fork first instead and then create a pull request, right 

The devcontainer guidelines tell to open the original repo, so if you make changes in your container and then commit you are not following the guidelines right?

Or am I wrong?

Reply:
yea if you're looking to contribute you definitely need a fork. Thought some folks may just want to run langchain in a container, but guess it's safe to assume most folks may want to contribute as well 👍